.golangci.yml

```diff
@@ -0,0 +1,38 @@
+service:
+  project-path: github.com/kubesaw/ksctl
+  prepare: # see https://github.com/golangci/golangci/wiki/Configuration#config-directives
+    - make generate
+
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 10m
+
+
+linters:
+  enable:
+    - megacheck
```

Comment:
Should we think about including `gosec` linter?

Reply:
yeah, why not: 1de3bf4